### PR TITLE
docs: add databend example

### PR DESCRIPTION
**Pull request by ZhiHanZ**

### CODE REVIEW

File: docs/examples/databend.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+In this document we'll show how to set up a simple data pipeline to load data from indexed.xyz to databend cloud and perform data analysis through SQL.
+
+Currently we are under Beta Access and You could easily take a trial on databend cloud thourgh the [link](https://app.databend.com/)
+
+## Load data from indexed.xyz
+
+Currently all files stored in indexed.xyz in under [cloudflare R2](https://www.cloudflare.com/products/r2/), and we are using prefix `9d` for the demo.
+
+At First, we need to add indexed.xyz R2 as a databend [stage](https://databend.rs/doc/sql-commands/ddl/stage/).
```
Comment (bohutang) [inline docs/examples/databend.md]:
```suggestion
At First, we need to add indexed.xyz R2 as a databend [stage](https://docs.databend.com/sql/sql-commands/ddl/stage/).
```

File: docs/examples/databend.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: Databend Cloud
+---
+
+In this document we'll show how to set up a simple data pipeline to load data from indexed.xyz to databend cloud and perform data analysis through SQL.
+
+Currently we are under Beta Access and You could easily take a trial on databend cloud thourgh the [link](https://app.databend.com/)
+
+## Load data from indexed.xyz
+
```
Comment (bohutang) [inline docs/examples/databend.md:10]: Better to add a explaination: such as all the queries are running in the worksheet, and worksheet doc is in https://docs.databend.com/using-databend-cloud/worksheet

File: docs/examples/databend.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+| 2022-05-01 | 426 |
+| 2022-03-01 | 415 |
+| 2021-04-01 | 48 |
+| 2021-09-01 | 257 |
+| 2023-02-01 | 1950 |
+| 2022-09-01 | 245 |
+
+### Visualize result through databend worksheet
+
+Databend Cloud natively provided a sort of visualization toolkits on worksheet.
```
Comment (flaneur2020) [inline docs/examples/databend.md:141]: how about clip the screen about how the visualization is configured ?

File: docs/examples/databend.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+| ---- | ---- | --- | ------------- | ------- |
+| dt=2015/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-0-4.parquet | 12616 | e273f913a5f75fea96f9fe7d2f086a6f | 2023-04-07 02:02:21.717 +0000 | NULL |
+| dt=2015/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-0-42.parquet | 5470 | a29c8427372d02dece62924dd985ae51 | 2023-04-07 02:02:21.725 +0000 | NULL |
+| dt=2016/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-37-26.parquet | 32708 | 2c21a9ffbec6d900f21fe4e8ff30d00f | 2023-04-07 02:02:22.131 +0000 | NULL |
+| dt=2016/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-37-27.parquet | 261112 | 8efed0a4490b52bb35a12eb1d2162d18 | 2023-04-07 02:02:21.724 +0000 | NULL |
+| dt=2016/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-37-44.parquet | 10927 | d5f7c980ac9e1e1c3de8d42c4388a7b1 | 2023-04-07 02:02:21.794 +0000 | NULL |
+| dt=2016/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-37-45.parquet | 273489 | e41f8268c73ff9e8062a16fd1bd3d8bf | 2023-04-07 02:02:22.180 +0000 | NULL |
+| dt=2016/1680119600-25ee0299-28eb-47a4-b81e-f49f5399617b-37-9.parquet | 4220496 | f206d6bcb48483dc684ba8c67d5c6733 | 2023-04-07 02:02:22.145 +0000 | NULL |
+
+### Creeate contract table
```
Comment (bohutang) [inline docs/examples/databend.md]:
```suggestion
### Create contract table
```

File: docs/examples/databend.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  `log_index` BIGINT NULL,
+  `transaction_hash` VARCHAR NULL,
+  `transaction_index` BIGINT NULL,
+  `data` VARCHAR NULL,
+  `topics` VARCHAR NULL,
+  `id` VARCHAR NULL
+);
+```
+
+### Load Data from stage
```
Comment (bohutang) [inline docs/examples/databend.md]:
```suggestion
### Load data from stage
```

File: docs/examples/databend.md (hunk 3 above)
Comment (flaneur2020) [reply docs/examples/databend.md:141]: like this one: img_v2_1dc1cacd-cd2e-485e-86ce-408bc30fa50g

this image gives more infomation about how to configure a chart like this

File: docs/examples/databend.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+| 2022-12-01 | 1566 |
+| 2022-01-01 | 272 |
+| 2022-05-01 | 426 |
+| 2022-03-01 | 415 |
+| 2021-04-01 | 48 |
+| 2021-09-01 | 257 |
+| 2023-02-01 | 1950 |
+| 2022-09-01 | 245 |
+
+### Visualize result through databend worksheet
```
Comment (bohutang) [inline docs/examples/databend.md]:
```suggestion
### Visualize result through Databend worksheet
```

File: docs/examples/databend.md (hunk 3 above)
Comment (ZhiHanZ) [reply docs/examples/databend.md:141]: done

File: docs/examples/databend.md (hunk 2 above)
Comment (ZhiHanZ) [reply docs/examples/databend.md:10]: Added explaination:
We would use databend [worksheet](https://docs.databend.com/using-databend-cloud/worksheet) to run all queries with the XSmall instance.

**Pull request merged**